lib/js/jsffi.nim

```diff
@@ -486,3 +486,35 @@ macro bindMethod*(procedure: typed): auto =
       newTree(nnkStmtList, thisQuote, call)
   )
   result = body
+
+
+
+proc generateJsObject(args: varargs[NimNode]): NimNode =
+  var properties = @args[0]
```

Comment:
What is the point of this line?

Reply:
Yeah, that wasn't very clear, I changed it to take just a NimNode directly in the signature in a commit